### ISSUE TITLE
Not working.  Just a black screen

### ISSUE THREAD

**Issue by ToysoftInc** [pedroSG94/Stream-USB-test#2]
I plugged in the usb camera and it ask to run the app but then it just show a black screen.
Selecting Start Stream does nothing.

I know the usb camera works.

**Comment by dbambini** [comment]
I have the same problem

**Comment by ToysoftInc** [comment]
The bug says cancel right after agreeing to the permission.

can somebody please help? thanks

**Comment by ashikxcel** [comment]
Did you add this line in manifest file

<uses-feature android:name="android.hardware.usb.host"/>

**Comment by ashikxcel** [comment]
> The bug says cancel right after agreeing to the permission.
> 
> can somebody please help? thanks

Did you add this line in manifest file

`<uses-feature android:name="android.hardware.usb.host"/>`

**Comment by ToysoftInc** [comment]
Yes. I did not modify the project and the permission is already there in the manifest file.
